src/index.ts

```diff
@@ -6,24 +6,14 @@ import { Transformer } from "./Transformer";
 
 const tsx = /\.tsx?$/;
 
-// Parse a TypeScript module into an ESTree program.
-function transformFile(input: ts.SourceFile): SourceDescription {
-  const transformer = new Transformer(input);
-  const { ast, fixups } = transformer.transform();
-
-  // NOTE(swatinem):
-  // hm, typescript generates `export default` without a declare,
-  // but rollup moves the `export default` to a different place, which leaves
-  // the function declaration without a `declare`.
-  // Well luckily both words have the same length, haha :-D
-  let code = input.getText();
-  code = code.replace(/(export\s+)default(\s+(function|class))/m, "$1declare$2");
-  for (const fixup of fixups) {
-    code = code.slice(0, fixup.range.start) + fixup.identifier + code.slice(fixup.range.end);
-  }
+const toArray = <T>(value: T): T extends ReadonlyArray<any> ? T : [T] =>
```

Comment:
```suggestion
function toArray<T>(value: T | ReadonlyArray<T>): Array<T> {
  return Array.isArray(value) ? value : [value];
}
```

Well this is just personal preference, but since I now have external contributors 🎉 , it might be a good idea to set up some linting:

* prefer function declarations over arrows on the toplevel
* avoid `any` as much as possible (you cast things a couple of times)
* prefer generics notation over `T[]` (you actually made this a one-element tuple, not sure if that was intentional?)

Reply:
> (you actually made this a one-element tuple, not sure if that was intentional?)

It was intentional, since we know the array only has one value when a non-array is passed, but it's largely inconsequential. :)